CODE_OF_CONDUCT.md

```diff
@@ -1,46 +1,9 @@
-# Contributor Covenant Code of Conduct
+Welcome to the Weblate community
 
-## Our Pledge
+Accept and offer criticism constructively.
+Let anyone have the privacy they desire.
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+Settle differences within these boundaries.
 
-## Our Standards
-
-Examples of behavior that contributes to creating a positive environment include:
-
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
```

Comment:
This is not qualified relating to an acceptance for diversity of thought. Insofar as it means all, no. Respect is given, not taken for granted. Issuing respect by default undermines the concept, unless used for a genuinely noble concept like freedom of speech.